include/swift/AST/Decl.h

```diff
@@ -4325,15 +4325,28 @@ class AbstractStorageDecl : public ValueDecl {
 
 /// VarDecl - 'var' and 'let' declarations.
 class VarDecl : public AbstractStorageDecl {
+public:
+  enum class Specifier : uint8_t {
+    // For Var Decls
+    
+    Let  = 0,
+    Var  = 1,
+    
+    // For Param Decls
+    
+    None  = Let,
```

Comment:
The default convention for let-bound vardecls and un-annotated parameters is `Owned` - and this was modeling the un-annotated part.  I can remove the alias and use `Owned` everywhere.